ExactSolutionEx1.cc

```diff
@@ -0,0 +1,46 @@
+#include "ExactSolutionEx1.hh"
+#include <cmath>
+// Example-1, (Taken from numerical experiments of paper Houston et al.)
+// The solution is smooth on L-shape domain in [-1,1]*[-1,1]\(0,1)*(-1,0)
+template <>
+void ExactSolutionEx1<2>::vector_value(const Point<2> &p, Vector<double> &values) const
+{
+  values(0) = -1.0 * std::exp(p(0)) * (p(1) * std::cos (p(1)) + std::sin (p(1)));
```

Comment:
The code would be more clear with ```using std::exp;```, ```using std::cost```, and ```using std::sin``` You should also groups the terms that go together, it's hard to see what's in brackets